Issue #4394: increase coverage of pitest-checkstyle-api-filters profile to 90%

Nimfadora opened the pull request:
Issue #4394

[pitest-checkstyle-api-filters profile before changes](https://nimfadora.github.io/issue4394.html)

increased mutation threshold for pitest-checkstyle-api-filters profile by 6%
current threshold: 90%
execution time: 18:23 min

Nimfadora commented:
Ok. 
I'm changing the line of code where mutation survives in the same way that pitest do it. Then I'm running all tests. It takes less that a minute for me. If some of them fail, I take the one of them and put to the targetTests

romani commented:
> If some of them fail, I take the one of them and put to the targetTests

ok, but please do copy of failed test method. Usage of whole test class is too much.

Nimfadora commented:
@romani I've been doing copy of failing methods, but realized - it is looks not so good as it seemed from the start. Quite often we need to copy tests with a number of private/inherited methods that called within test. Like `verify`, `getCheckMessage`,` createCheckConfig` which Checks UTs inherit from BaseCheckTestSupport.
Instead, I reviewed targetTest that I've added to api-filters profile, and realized, that half of them are redundant. Looks like mutations caused failure of a bunch of tests and I added new target test disregurding whether any of failed tests are already in the list.
So, with removing of the half of targetTests I was able to reduce exectution time to `04:06 min`. May be it fine to leave it as it is?

romani commented:
>  May be it fine to leave it as it is?

ok, for now.

romani commented:
on my local your branch is still `[INFO] Total time: 09:46 min`

Nimfadora commented:
Please, recheck it again. I made my changes to improve performance only today.

romani commented:
I do not see commit reference after my commit, did you push your changes ?
if you did "push ..... --force" commit reference in page usually goes down.

Nimfadora commented:
@romani sorry, some github bug, It was linked to the issue but not to the PR, repushed.

romani commented:
on my local it is `[INFO] Total time: 03:34 min` now.
